src/domain/chains/entities/gas-price-oracle.entity.ts

```diff
@@ -0,0 +1,8 @@
+import { GasPrice } from './gas-price.entity';
+
+export interface GasPriceOracle extends GasPrice {
+  type: 'oracle';
+  uri: string;
+  gasParameter: string;
+  gweiFactor: number;
```

Comment:
Maybe this was an issue in the cgw but `gweiFactor` is a string in the current cgw.` gwei_factor: String`

Reply:
I will keep it as `string` since we don't perform any numeric manipulation and we need to evaluate the usage of a big decimal numbers in the CGW.

Reply:
Done in [34d40f2](https://github.com/5afe/safe-client-gateway-nest/pull/93/commits/34d40f226f67c3b006a3f1eccccfe9dc022ad746)